tests/test_TFNetworkLayer.py

```diff
@@ -8132,6 +8132,23 @@ def test_PostfixInTimeLayer():
         assert out[0, src_seq_lens[0] + repeat - 1, 0] == -7
 
 
+def test_FakeQuantizeStaticLayer():
+  with make_scope() as session:
+    num_inputs = 3
+    config = Config()
+    config.update({
+      "extern_data": {"data": {"dim": num_inputs}},
+      "network": {
+        "output": {"class": "fake_quantize_static", "range_min": -6, "range_max":6, "num_bits":2, "from": "data"}
+      }})
+    network = TFNetwork(config=config, train_flag=True)
+    network.construct_from_dict(config.typed_value("network"))
+    feed = {network.extern_data.get_data("data").placeholder: numpy.asarray([[[-5,0,3]]])}
```

Comment:
I would rather use the `make_feed_dict` function, which is more correct. (You miss many things here, like the sizes, etc.)

Reply:
I did not want to use `make_feed_dict` (I had this in the beginning) because it does not allow me to put manual values.